mmdet/models/dense_heads/fcos_head.py

```diff
@@ -219,6 +219,7 @@ def loss(self,
         pos_inds = ((flatten_labels >= 0)
                     & (flatten_labels < bg_class_ind)).nonzero().reshape(-1)
         num_pos = len(pos_inds)
+        num_pos = max(reduce_mean(torch.tensor(num_pos).cuda()).item(), 1.0)
         loss_cls = self.loss_cls(
             flatten_cls_scores, flatten_labels,
             avg_factor=num_pos + num_imgs)  # avoid num_pos is 0
```

Comment:
`# avoid num_pos is 0` becomes inconsistent with the code.
Is ` + num_imgs` still needed for AP?